lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
+
```

Comment:
It feels like this class is taking responsibility for two different things - managing the menu and displaying the menu. This is a bit nitpicky, but I'm going to pick nits because actually you've done a good job with the rest of the object model. 

Your method 'list' introduces some display logic around name and price, but a cleaner pattern here would be to have a MenuView or MenuFormatter class that did this for you and have Menu just responsible for the business logic. Something like:

```
class MenuView
  def initialise(menu)
    @menu = menu
  end

  def list
    @menu.getDishes.map { |name, price| "#{name}: £#{price}" }
  end
end
```

It's probably not evident here what advantages that has - I imagine in feels like just another class and a fairly pointless distinction - but as you start to deal with more complex interfaces you'll start to see the benefit of treating your view rendering logic as a separate concern to your data. 

(This is a tip to remember when you start the bowling challenge after week 5).